it/mail/chrome/messenger/messengercompose/composeMsgs.properties

```diff
@@ -56,94 +56,94 @@
 12505=Caricamento allegato in corso…
 
 ## @name NS_MSG_COULDNT_OPEN_FCC_FOLDER
-12506=Impossibile aprire la cartella Posta inviata. Verificare che le impostazioni di posta siano corrette.
+12506=Impossibile aprire la cartella Posta inviata. Verifica che le tue impostazioni di posta siano corrette.
 
 ## @name NS_MSG_CANT_POST_TO_MULTIPLE_NEWS_HOSTS
-12507=Non è possibile inviare verso più server di gruppi di discussione. Riprovare.
+12507=Non è possibile pubblicare su host di notizie multipli. Riprova.
```

Comment:
CORRECTIONS NEEDED. It should be as follows: 

+12507=Non è possibile pubblicare su host di newsletter. Riprova.